locales/en-US/pocket.properties

```diff
@@ -26,6 +26,8 @@ tagline = Save articles and videos from Firefox to view in Pocket on any device,
 taglinestory_one = Click the Pocket Button to save any article, video or page from Firefox.
 taglinestory_two = View in Pocket on any device, any time.
 tagssaved = Tags Added
+tos = By continuing, you agree to Pocket's *Terms of Service* and *Privacy Policy*
```

Comment:
Sorry for being a pain, but how are you going to distinguish the first link from the second? What if a localization swap the other for any reason? I strongly suggest to use something Mozilla localizers are used to, like `#1` and `#2`.

@mixedpuppy What were the other strings you were talking about on IRC?